setup/initial/setup-utils.sh

```diff
@@ -147,6 +148,21 @@ EOF
     amazon-cloudwatch-agent-ctl -a start
 }
 
+# Uses AWS_DOMAIN, SCRIPTS_PATH variables
+function update_motd() {
+    if [ -z "$AWS_DOMAIN" ]; then
+        return 0
+    fi
+
+    if [ "$AWS_DOMAIN" != "ov.internal" ]; then
+        # Ignore 99-warning from match files to copy
```

Comment:
I think we should install all mot.d files regardless aws_domain value. If other customers will use scripts in their aws accounts they may fork and customize banner and warnings

Reply:
@KeyMasterOfGozer agree?

Reply:
Maybe we should keep an easily modified section of the script with OneVizion specific things that a customer could modify for their needs?  The customer may have their own standards for these type of things.

Reply:
I think, these scripts cannot be used as is by other customers anyway, but can be a good starting point. We provide good practices, like SOC2 compliant warning, which customer can modify among other things by forking our public repository

Reply:
Also, when starting instances in dev account, I'm not using AWS_DOMAIN, but I still want warning to be adde

Reply:
@KeyMasterOfGozer, we have to agree on changes and complete review to move forward

Reply:
Sure, it's fine.